commit d69a15a164d3d0938dc55ef8f24f9062ab0984a7
Author: Mimi Zohar <zohar@linux.vnet.ibm.com>
Date:   Tue Jan 17 06:45:41 2017 -0500

    ima: fix ima_d_path() possible race with rename
    
    commit bc15ed663e7e53ee4dc3e60f8d09c93a0528c694 upstream.
    
    On failure to return a pathname from ima_d_path(), a pointer to
    dname is returned, which is subsequently used in the IMA measurement
    list, the IMA audit records, and other audit logging.  Saving the
    pointer to dname for later use has the potential to race with rename.
    
    Intead of returning a pointer to dname on failure, this patch returns
    a pointer to a copy of the filename.
    
    Reported-by: Al Viro <viro@zeniv.linux.org.uk>
    Signed-off-by: Mimi Zohar <zohar@linux.vnet.ibm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1259_d69a15a164d3d0938dc55ef8f24f9062ab0984a7.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1259_d69a15a164d3d0938dc55ef8f24f9062ab0984a7.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
